site/src/scaffold/token.html

```diff
@@ -54,8 +54,8 @@
 							</svg>
 						</span>
 				</button>
-				<a class="button button-clear" href="{{githubLink}}">
-					<span class="icon">
+				<a class="button button-clear" href="{{githubLink}}" aria-label="Link to Github">
```

Comment:
`Link to` is redundant, as the screenreader will announce `Link, Link to GitHub`. Additionally, it'd pose extra work for voice control users to determine that "Link to" is part of the name they'd have to say. I'd recommend just "GitHub."
```suggestion
				<a class="button button-clear" href="{{githubLink}}" aria-label="GitHub">
```

Reply:
Updated!